deploy/charts/cert-manager/values.yaml

```diff
@@ -83,11 +83,9 @@ serviceAccount:
   # Automount API credentials for a Service Account.
   automountServiceAccountToken: true
 
-# Optional additional arguments
+# Additional command line flags to pass to cert-manager controller binary.
+# To see all available flags run docker run quay.io/jetstack/cert-manager-controller:<version> --help
```

Comment:
Love the inclusion of "docker run quay.io/jetstack/cert-manager-controller:<version> --help"!

Reply:
Thanks 😄